client/src/dbus/algorithm.rs

```diff
@@ -20,11 +20,24 @@ impl Serialize for Algorithm {
         S: serde::Serializer,
```

Comment:
The changes on this file can be pulled out of this PR and be merged already

Reply:
Done, thanks!